packaging/nuget/NServiceBus.DataBus.AzureBlobStorage.nuspec

```diff
@@ -14,7 +14,7 @@
     <copyright>$copyright$</copyright>
     <tags>$tags$</tags>
     <dependencies>
-      <dependency id="WindowsAzure.Storage" version="[7.0.0, 8.0.0)" />
+      <dependency id="WindowsAzure.Storage" version="[8.0.0, 9.0.0)" />
```

Comment:
According to @Particular/azure-maintainers discussion, this might be a subject to change and go to
`[7.0.0, 9.0.0)`